### PR TITLE
OCR Bugfix + Multiprocessing Overhaul

### PR DESCRIPTION
**Pull request by ashermuse**
Originally just a bug fix to check if a document has a text layer page-by-page and then reOCR documents if they had a page with missing text.

Features:

- Multiprocessing reOCR of documents -- very fast checking of documents, OCR process is slower (takes about 30 seconds if no OCR needs to be done to scan 100 documents)
- Upped the batch size of the actual processing from 5 to 100, which should more than make up for OCR time (processing takes about 20% of what it took originally)
- Minor bug fixes to get topic models working

### CODE REVIEW

File: common/document_parser/process.py [hunk 1]
```diff
@@ -178,7 +185,61 @@ def process_dir(
...
+        # using list comprehension
+        process_list = [data_inputs[i * batch_size:(i + 1) * batch_size] for i in range((len(data_inputs) + batch_size - 1) // batch_size)]
+        
+        total_num_files = len(data_inputs)
+        reocr_count = 0
+        for item_process in tqdm(process_list):
+            with concurrent.futures.ProcessPoolExecutor(max_workers=5) as executor:
+                results = [executor.submit(check_ocr_status_job_type, data[1]) for data in item_process]
+                index = 0
+                for fut in concurrent.futures.as_completed(results):
```
Comment (takao8) [inline common/document_parser/process.py:205]: might want to do `for index, fut in concurrent.futures.as_completed(results): ` so you don't have to do the index+=1 iteration

Comment (ashermuse) [reply common/document_parser/process.py]: Good thought. Made that change.

**Comment by takao8** [comment]
Before approving and merging, we need to just make sure that the appropriate topic model is in production

**Comment by kroop-olivia-bah** [comment]
> Before approving and merging, we need to just make sure that the appropriate topic model is in production (bronze/gamechanger/models/topic_model/v2/topic_model_20221129162954.tar.gz)
> 
> As a follow-up, can we use the v4 topic models? @cWren0110 @kroop-olivia-bah

I'm not sure what the status of that model is, so I'll leave it to Ethan to answer

**Review by takao8 — APPROVED** [review]

**Review by takao8 — APPROVED** [review]

**Pull request merged**